source/user_defined_output/log_response_headers_plugin.cc

```diff
@@ -128,13 +122,16 @@ Envoy::ProtobufTypes::MessagePtr LogResponseHeadersPluginFactory::createEmptyCon
   return std::make_unique<LogResponseHeadersConfig>();
 }
 
-UserDefinedOutputPluginPtr LogResponseHeadersPluginFactory::createUserDefinedOutputPlugin(
-    const Envoy::Protobuf::Message& message, const WorkerMetadata& worker_metadata) {
-  const auto& any = dynamic_cast<const Envoy::ProtobufWkt::Any&>(message);
+absl::StatusOr<UserDefinedOutputPluginPtr>
+LogResponseHeadersPluginFactory::createUserDefinedOutputPlugin(
+    const Envoy::ProtobufWkt::Any& message, const WorkerMetadata& worker_metadata) {
   LogResponseHeadersConfig config;
+  Envoy::MessageUtil::unpackTo(message, config);
```

Comment:
Would `MessageUtil::unpackToNoThrow` be a better match now, since it returns status instead of throwing?